Resources/doc/guide.rst

```diff
@@ -0,0 +1,211 @@
+Beginners guide
+===============
+
+Befor starting this, you should have already installed:
+
+Text editor (Phpstorm, Vim, SublimeText, BlueFish..),
+
+Symfony,
+
+Composer,
+
+Git,
+
+ElasticSearch (and launched it, so you can go to localhost:9200 and see it's working)
+
+Create new Symfony project
+--------------------------
+Open Terminal, go to your working directory or create new
+
+.. code:: bash
+
+    mkdir Sites
+    cd Sites
+
+Create new Symfony project
+
+.. code:: bash
+
+    symfony new project
```

Comment:
This `symfony` command is available by using https://github.com/symfony/symfony-installer. It is recommended way but also mention composer `composer create-project symfony/framework-standard-edition`